tests/unit/Mvc/ModelTest.php

```diff
@@ -754,4 +755,51 @@ function () {
             }
         );
     }
+
+    public function testModelsCacheSnapshot(UnitTester $I)
+    {
+        if (!extension_loaded('apc')) {
+            $this->markTestSkipped(
+                'Warning: apc extension is not loaded'
+            );
+        }
+
+        if (!ini_get('apc.enabled') || (PHP_SAPI === 'cli' && !ini_get('apc.enable_cli'))) {
+            $this->markTestSkipped(
+                'Warning: apc.enable_cli must be set to "On"'
+            );
+        }
+
+        if (extension_loaded('apcu') && version_compare(phpversion('apcu'), '5.1.6', '=')) {
+            throw new \PHPUnit_Framework_SkippedTestError(
+                'Warning: APCu v5.1.6 was broken. See: https://github.com/krakjoe/apcu/issues/203'
+            );
+        }
+
+        $I->haveServiceInDi(
```

Comment:
could you please use `addServiceToContainer` insted of `haveServiceInDi` to avoid future refactoring